### PR TITLE
mutable_immutable

### PR DESCRIPTION
**Pull request by Ihor-MA**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -16,4 +16,18 @@
...
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+
+sorted_variables = {
+    "mutable": [],
+    "immutable": []
+}
+
+list_of_variables = [lucky_number, pi, one_is_a_prime_number, name,
```
Comment (Kroll410) [inline app/main.py:25]: No calculation needed, only pass variables in the correct lists

**Review by Kroll410 — CHANGES_REQUESTED** [review]

**Review by Y-Havryliv — CHANGES_REQUESTED** [review]
The solution is good, but you pushed redundant files. Please delete them.

**Comment by Kroll410** [comment]
Python cache files are still there

**Review by Kroll410 — CHANGES_REQUESTED** [review]
Remove redundant files

**Comment by Ihor-MA** [comment]
> Remove redundant files



> Remove redundant files

**Pull request closed without merging**